src/main/scala/com/databricks/sparkdl/python/PythonInterface.scala

```diff
@@ -43,8 +44,18 @@ private var _sqlCtx: SQLContext = null
   def listToVectorFunction(col: Column): Column = {
     Conversions.convertToVector(col)
   }
+
+  /**
+   * Create an UDF as the result of chainning multiple UDFs
+   */
+  def registerPipeline(name: String, udfNames: Seq[String]) = {
```

Comment:
as mentioned, it is better to keep all the interfaces between python and scala to simple things like `ArrayList`, this simplifies the maintenance.